assets/js/admin/settings/tax-rates/views/table-add.js

```diff
@@ -166,12 +166,24 @@ const TableAdd = wp.Backbone.View.extend( {
 			i18n,
 		} = eddTaxRates;
 
+		if ( ! this.model.get( 'country' ) ) {
+			alert( i18n.emptyCountry );
+
+			return;
+		}
+
 		if ( existingCountryWide.length > 0 ) {
 			alert( i18n.multipleCountryWide.replace( '%s', this.model.get( 'country' ) ) );
 
 			return;
 		}
 
+		if ( this.model.get('amount') <= 0 ) {
```

Comment:
```suggestion
		if ( this.model.get( 'amount' ) <= 0 ) {
```

Just some space around the parameter. :)